src/components/Composer.vue

```diff
@@ -342,7 +342,9 @@ export default {
 					bcc: this.selectBcc.map(this.recipientToRfc822).join(', '),
 					draftUID: uid,
 					subject: this.subjectVal,
-					body: this.editorPlainText ? htmlToText(this.bodyVal) : this.bodyVal,
+					body: this.editorPlainText
```

Comment:
Works, but from a logical PoV the composer shouldn't have to care about this. This should be moved to the `TextEditor.vue`

Reply:
Makes sense but I can't figure out how to implement this there.

What could probably easily be done would be to add a "style" attribute to all `<p>` elements rather than assign them a class:

`viewWriter.addClass('null-margin', conversionApi.mapper.toViewElement(data.item))`

might be changed to add an attribute rather than adding a class.

Reply:
> Makes sense but I can't figure out how to implement this there.

Possibly at https://github.com/nextcloud/mail/blob/1691463ecdfa207f5e470c7592196812b76ff563/src/components/TextEditor.vue#L141 but CKEditor might remove the `style` tag. I also just realized that when we only add the style element when the message is sent, the previewed message looks different than what is actually sent.



> What could probably easily be done would be to add a "style" attribute to all `<p>` elements rather than assign them a class:

If that is easy to do I'm fine with that as well :) This will also show a correct preview of what will be sent (independent of Nextcloud's paragraph styling)